src/effects/builtin/CMakeLists.txt

```diff
@@ -68,6 +68,14 @@ set(AU3_SRC
     ${AU3_LIBRARIES}/lib-builtin-effects/ToneGenBase.h
     ${AU3_LIBRARIES}/lib-builtin-effects/Generator.cpp
     ${AU3_LIBRARIES}/lib-builtin-effects/Generator.h
+    ${AU3_LIBRARIES}/lib-builtin-effects/Fade.cpp
+    ${AU3_LIBRARIES}/lib-builtin-effects/Fade.h
+    ${AU3_LIBRARIES}/lib-builtin-effects/Invert.cpp
+    ${AU3_LIBRARIES}/lib-builtin-effects/Invert.h
+    ${AU3_LIBRARIES}/lib-builtin-effects/Repair.cpp
+    ${AU3_LIBRARIES}/lib-builtin-effects/Repair.h
+    ${AU3_LIBRARIES}/lib-builtin-effects/Reverse.cpp
+    ${AU3_LIBRARIES}/lib-builtin-effects/Reverse.h
```

Comment:
I think it would be better to move these effects to AU4, like the others... we can put them in one directory, or in different ones...

Reply:
Yes, can do that.